config/locales/simple_form.en.yml

```diff
@@ -205,6 +205,7 @@ en:
         setting_expand_spoilers: Always expand posts marked with content warnings
         setting_hide_network: Hide your social graph
         setting_noindex: Opt-out of search engine indexing
+        setting_norss: Opt-out of an RSS feed for your public posts
```

Comment:
I think the wording is unclear, unless you know that having an RSS feed is even a thing and what that even means.
I'm not sure what the best alternative would be though.